src/loaders/JsonImageInfoLoader.ts

```diff
@@ -136,7 +136,7 @@ class JsonImageInfoLoader implements IVolumeLoader {
     const imageInfo = convertImageInfo(this.jsonInfo);
 
     this.cacheStore = this.cache?.addVolume(imageInfo.numChannels, Math.max(imageInfo.times, this.urls.length), [
-      new Vector3(imageInfo.subregionSize.x, imageInfo.subregionSize.y, imageInfo.subregionSize.z),
+      new Vector3(1, 1, 1),
```

Comment:
Why does this get converted to 1,1,1 here? Is it because chunks don't need to be scaled, or...?